src/main/resources/site/layouts/one-column/one-column.js

```diff
@@ -1,18 +1,16 @@
-var stk = require('/lib/stk/stk');
-var portal = require('/lib/xp/portal');
+const stk = require('/lib/stk/stk');
+const portal = require('/lib/xp/portal');
 
-exports.get = handleGet;
+exports.get = function(req) {
 
-function handleGet(req) {
+    const component = portal.getComponent();
 
-    var component = portal.getComponent();
-
-    var params = {
+    const params = {
         component: component,
         leftRegion: component.regions["left"]
     };
 
-    var view = resolve('one-column.html');
+    const view = resolve('one-column.html');
```

Comment:
Move the view outside the function. Don't need to resolve it each time.

Reply:
Agree. I thought I'd improve these other js files (other than default.js) in separate PRs, this PR only focuses on `var`.